### PR TITLE
Add FakeItEasy to projects using GitLink

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ Below is a list of projects already using GitLink (alphabetically ordered).
 
 - <a href="http://www.catelproject.com" target="_blank">Catel</a>
 - <a href="http://www.expandframework.com/" target="_blank">eXpand</a>
+- <a href="https://github.com/FakeItEasy/FakeItEasy" target="_blank">FakeItEasy</a>
```
Comment (blairconrad) [inline README.md]: @adamralph, why the github repo instead of the web site?

Comment (adamralph) [inline README.md]: @bradwilson good point. Just didn't think. I'll change it.

Comment (gep13) [inline README.md]: @adamralph let me know when you raise the PR, and I can merge it for you.

Comment (adamralph) [inline README.md]: https://github.com/GitTools/GitLink/pull/92

Comment (blairconrad) [inline README.md]: Thanks, sirs.

**Pull request merged**